MANIFEST.in

```diff
@@ -1,7 +1,16 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/zope-product
 include *.rst
 include *.txt
+include buildout.cfg
+include tox.ini
 
-recursive-include src *
-
-global-exclude *.pyc
-global-exclude *.pyo
+recursive-include src *.pt
```

Comment:
There's so much stuff here by default, are you working off an older version of the `meta` package?

Reply:
I was wondering, too so I checked whether I am on the current tip of `master`.
The after checking the `meta` repository here on GitHub just now I found out that https://github.com/zopefoundation/meta/pull/86 is not merged yet.